.github/PULL_REQUEST_TEMPLATE/release_template.md

```
@@ -0,0 +1,16 @@
+### Release Request
+
+#### JIRA ID
+<!-- Provide a detailed description of the feature -->
+
+#### Module
+<!-- Provide a detailed description of the feature -->
+
+#### Description
+<!-- Provide a detailed description of the feature -->
+
+#### List of Features 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)